cmake/inference_lib.cmake

```diff
@@ -150,6 +150,36 @@ endfunction()
 set(inference_lib_deps third_party paddle_fluid paddle_fluid_c paddle_fluid_shared paddle_fluid_c_shared)
 add_custom_target(inference_lib_dist DEPENDS ${inference_lib_deps})
 
+set(dst_dir "${FLUID_INFERENCE_INSTALL_DIR}/third_party/eigen3")
```

Comment:
why do you add these dependencies?

Reply:
Because some conflicts of older commits, will change it